common/cards/base/card.ts

```diff
@@ -27,12 +28,16 @@ abstract class Card {
 	public name: string
 	public rarity: CardRarityT
 
+	/** The battle log attached to this attack */
```

Comment:
This should say attached to this *card*. Also could I get clarification, ideally in a comment here, when this is used and what the value should be set as

Reply:
👍